### DISCUSSION TITLE
Does vweb support layouts as well?

### DISCUSSION THREAD

**Discussion by GildedHonour** [vlang/v#13532] [General]
Does vweb support layouts as well? Is there an example?

**Comment by JalonSolov** [comment]
Do you have an example of what you're asking about?

**Reply by ITwrx** [reply]
nested templates, where one template (commonly referred to as a `layout`) is used by many child templates and contains the code that is common to the child templates that use that template/layout. For example, one may use the `main_layout` template for the `header`, `top_nav`, `breadcrumbs` and `footer` for the site front end, and the `admin_layout` for the admin section of the application. The `admin_layout` might have different `top_nav`, no `breadcrumbs` and different `footer`/no `footer`.

I haven't looked to see if this is supported yet. Just figured i'd try to answer the question, in general.
